driver/flambda_backend_args.ml

```diff
@@ -22,6 +22,9 @@ let mk_flambda2_debug f =
 let mk_no_flambda2_debug f =
   "-no-flambda2-debug", Arg.Unit f, " Disable debug output for the Flambda2 pass"
 
+let mk_no_mach_ir f =
+  "-no-mach-ir", Arg.Unit f, " Avoid using the Mach IR"
```

Comment:
Maybe update the string to indicate this is a no-op kept for compatibility reasons?

Reply:
Good idea, thanks for the review.